R/backport_linter.R

```diff
@@ -90,7 +90,7 @@ normalize_r_version <- function(r_version) {
     version_names <- c("devel", "release", paste0("oldrel-", seq_len(length(minor_versions) - 2L)))
     if (!r_version %in% version_names) {
       # This can only trip if e.g. oldrel-99 is requested
-      stop("`r_version` must be a version number or one of ", toString(sQuote(version_names)), call. = FALSE)
+      cli::cli_abort("{.var r_version} must be a version number or one of {toString(sQuote(version_names))}")
```

Comment:
Use `{.val {sQuote(version_names)}}` instead? Is there a {cli}-specific way to do `sQuote()` too?

https://cli.r-lib.org/articles/semantic-cli.html#inline-text-formatting